service/internal/cloudconfig/template/decrypt_tls_assets_script.go

```diff
@@ -33,7 +33,9 @@ docker run --net=host -v /etc/kubernetes/ssl:/etc/kubernetes/ssl \
 
 main() {
   kms_tls_assets_decrypt
-  chown -R etcd:etcd /etc/kubernetes/ssl/etcd
+  if [ -d "/etc/kubernetes/ssl/etcd" ]; then 
+    chown -R etcd:etcd /etc/kubernetes/ssl/etcd
+  fi
```

Comment:
in the new version, there is no more `/etcd` folder on worker nodes so making the condition to avoid failures on systemd unit

Reply:
Should we have a TODO issue to be reminded of cleaning this up in the next major version?

Reply:
We cannot clean this. On master nodes, we still have that dir a  for security reason need to change the ownership. 

We use the same  unit for master/workers so this needs to stay for now. IMHO Creating separate units o for master nad workers would be overkill.